doc/api/stream.md

```diff
@@ -2150,8 +2150,9 @@ class SourceWrapper extends Readable {
 }
 ```
 
-The `readable.push()` method is intended be called only by `Readable`
-implementers, and only from within the `readable._read()` method.
+The `readable.push()` method is used to push the content
+into the internal buffer. It is recommended that the `readable.push()` method
```

Comment:
Is it better to remove `It is recommended that`?

```suggestion
into the internal buffer. The `readable.push()` method
```

(Probably change `could ` to `can` on the next line if we make this suggested change.)

Reply:
Yep. It's fine.